### DISCUSSION TITLE
Logon to TikTok "too many attempts" error.

### DISCUSSION THREAD

**Discussion by jwalway** [cefsharp/CefSharp#4128] [General]
Hello,

I'm not sure if this is the place to ask, but I'm curious about an issue that arose when I (and another person) tried browsing with the CefSharp.MinimalExample.Wpf.netcore browser.   I tried logging onto my TikTok account and it came up with the error "Too many attempts."    I was finally able to login, but my friend is having the same problem and he can find no way around it.   He can login fine with the standard chrome browser.

I compiled all of this in Visual Studio using .Net 6.0.   Any idea what this could be?  

Thanks for any help!

**Comment by amaitland** [comment]
It's not uncommon for high profile websites to specifically block embedded browsers, so that's something to keep in mind. There are a number of debugging steps you can try.

- You can check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file
- You can also try disabling CEF request interception. 

```
settings.CefCommandLineArgs.Add("disable-request-handling-for-testing")
```

https://github.com/cefsharp/CefSharp/wiki/General-Usage#cefsettings-and-browsersettings

- Trying a different user agent might be worth trying also

Your friend might be blocked and has to wait a period of time before attempting another login attempt.

**Reply by jwalway** [reply]
Hello amaitland,

Thanks much for the reply!   I'll look into those things.
